Doc/library/stdtypes.rst

```diff
@@ -1406,12 +1406,21 @@ The following methods are present only on unicode objects:
    that have the Unicode numeric value property, e.g. U+2155,
    VULGAR FRACTION ONE FIFTH.
 
+.. method:: unicode.isdigit()
+
+   Return ``True`` if there are only digit characters in S, ``False``
```

Comment:
This method doesn't refer to `S` anywhere else, presumably the "in S" thing can be replaced with "in the string" or "in the unicode string"? 

I imagine you can just copy the exact wording from [the 3.7 documentation](https://docs.python.org/3.7/library/stdtypes.html#str.isdigit) (seems like that's mostly what this is, but with the "in S" thing for consistency with the other methods).